tests/conformance/test_conformance.py

```diff
@@ -523,7 +523,7 @@ def bucket(client):
 @pytest.fixture
 def object(client, bucket):
     blob = client.bucket(bucket.name).blob(uuid.uuid4().hex)
-    blob.upload_from_string(_STRING_CONTENT, checksum="crc32c")
+    blob.upload_from_string(_STRING_CONTENT)
```

Comment:
This change doesn't seem aligned with the PR description.  Is there some reason we are using the default (`"md5"`) rather than an explicity `"crc32c"`?

Reply:
Sorry I slipped in an irrelevant change to the PR. This was a workaround for a testbench emulator issue. The issue is now resolved in the testbench emulator. Replacing it back to default values as I'm aiming to use as many default values as possible in the conf tests.